plugins/source/gcp/resources/plugin/tables.go

```diff
@@ -54,28 +54,28 @@ import (
 
 func PluginAutoGeneratedTables() schema.Tables {
 	tables := []*schema.Table{
-		aiplatform.JobLocations(),
```

Comment:
Can we do this change in a separate PR? Also this is a user facing change as it impacts the order of how tables are synced so should probably be a `fix`

Reply:
reverted in https://github.com/cloudquery/cloudquery/pull/15783/commits/0e8d9b0cf065b18f5905f908bcfc3659bad886b1